backend/store/instance.go

```diff
@@ -211,27 +213,35 @@ func (s *Store) CreateInstanceV2(ctx context.Context, instanceCreate *InstanceMe
 		return nil, common.Errorf(common.NotFound, "environment %s not found", instanceCreate.EnvironmentID)
 	}
 
+	where := ""
+	if instanceCreate.Activation {
```

Comment:
Instead of implicitly calculating activation, maybe we can run a query and return an error if activation == true and count > limit?

Reply:
Sure, we did this in the api server https://github.com/bytebase/bytebase/pull/6714/files#diff-31ab5c1bfbd5317fd0516a36cb491f0f9c2731b5c54de159500ae6c6a3b4c480R121, but I'm considering to add this limit for atomicity